### PR TITLE
Translate Istio 1.0 canary into Chinese

### PR DESCRIPTION
**Pull request by DavadDi**

### CODE REVIEW

**Comment by rootsongjc** [comment]
@DavadDi Please sign the CLA first.

File: content_zh/blog/2017/0.1-canary/index.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+## Kubernetes 中的金丝雀部署
+
+假设我们有一个已部署的 **helloworld** 服务 **v1** 版本，我们想要测试（或简单上线）新版本 **v2**。 使用 Kubernetes，您可以通过简单地更新服务的 [Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) 中的镜像并自动进行部署来[上线](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#updating-a-deployment)新版本的 **helloworld** 服务。 如果我们特能够小心保证在启动并且在仅启动一个或两个 v2 副本[暂停](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#pausing-and-resuming-a-deployment)上线时有足够的 **v1** 副本运行，则能够保持金丝雀发布对系统的影响非常小。 后续我们可以观察效果，或在必要时进行[回滚](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#rolling-back-a-deployment)。 最好，我们也能够对 Deployment 设置 [HPA](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#scaling-a-deployment)，在上线过程中减少或增加副本以处理流量负载时，也能够保持副本比例一致。
+
+尽管这种机制能够很好工作，但这种方式只适用于部署的经过适当测试的版本，也就是说，更多的是蓝/绿发布，又称红/黑发布，而不是 “蜻蜓点水“ 式的金丝雀部署。 实际上，对于后者（例如，并没有完全准备好或者无意对外暴露的版本），Kubernetes 中的金丝雀部署将使用具有[公共 pod 标签](**https://kubernetes.io/docs/concepts/cluster-administration/manage-deployment/#using-labels-effectively**)的两个 Deployment 来完成。 在这种情况下，我们不能再使用自动缩放器，因为是有由两个独立的自动缩放器来进行控制，不同负载情况下，副本比例（百分比）可能与所需的比例不同。
```
Comment (fleeto) [inline content_zh/blog/2017/0.1-canary/index.md]: 链接语法？

**Comment by DavadDi** [comment]
I signed it!

File: content_zh/blog/2017/0.1-canary/index.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+
+无论我们使用一个或者两个部署，使用 Docker，Mesos/Marathon 或 Kubernetes 等容器编排平台进行的金丝雀发布管理都存在一个根本问题：使用实例扩容来管理流量；版本流量分发和副本部署在上述平台中并独立。 所有 pod 副本，无论版本如何，在 kube-proxy 循环池中都被一视同仁地对待，因此管理特定版本接收的流量的唯一方法是控制副本比例。 以小百分比维持金丝雀流量需要许多副本（例如，1％ 将需要至少 100 个副本）。 即使我们可以忽略这个问题，部署方式功能仍然非常有限，因为它只支持简单（随机百分比）金丝雀部署。 如果我们想根据某些特定规则将请求路由到金丝雀版本上，我们仍然需要另一种解决方案。
+
+## 使用 Istio
+
+使用 Istio，流量路由和副本部署是两个完全独立的功能。 服务的 pod 数量可以根据流量负载灵活伸缩，与版本流量路由的控制完全正交。 这在自动缩放的情况下能够更加简单地管理金丝雀版本。 事实上，自动缩放管理器仍然独立运行，其在响应因流量路由导致的负载变化与其他原因导致负载变化的行为上没有区别。
+
+Istio 的[路由规则](/docs/concepts/traffic-management/#rule-configuration)也带来了其他的便利；你可以轻松实现细粒度控制流量百分比（例如，路由 1％ 的流量而不需要 100 个 pod），当然也可以使用其他规则来控制流量（例如，将特定用户的流量路由到金丝雀版本）。 作为展示，让我们看一下采用这种方式部署 **helloworld** 服务的简单便捷。
```
Comment (fleeto) [inline content_zh/blog/2017/0.1-canary/index.md]: 中文版链接

File: content_zh/blog/2017/0.1-canary/index.md (hunk 1 above)
Comment (DavadDi) [reply content_zh/blog/2017/0.1-canary/index.md]:
```
[公共 pod 标签](**https://kubernetes.io/docs/concepts/cluster-administration/manage-deployment/#using-labels-effectively**)

这个吗？
```

File: content_zh/blog/2017/0.1-canary/index.md (hunk 2 above)
Comment (DavadDi) [reply content_zh/blog/2017/0.1-canary/index.md]: ok，我重新修改一下

**Pull request closed without merging**